js/bundle/src/cli.ts

```diff
@@ -110,6 +144,15 @@ export function main() {
       ? DEFAULT_VERSION
       : options.formatVersion;
 
+  const headerOverrides =
+    options.headerOverride === undefined
+      ? undefined
+      : JSON.parse(fs.readFileSync(options.headerOverride, 'utf8'));
```

Comment:
Maybe try/catch the JSON.parse and throw a custom error message saying that the header overrides are invalid JSON?